src/Assignments/SimpleProblemPage.tsx

```diff
@@ -114,25 +321,76 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
         return <div>{error}</div>;
     }
 
-    if (_.isEmpty(problems)) return <div>There was an error loading this assignment.</div>;
+    // if (_.isEmpty(problems)) return <div>There was an error loading this assignment.</div>;
```

Comment:
Not a bad point, an empty array does not necessarily mean there is an error, however, I wonder how this impacts the rest of the application

Reply:
@doombot-exe is the best

Reply:
handled